downloads/swan-lake-release-notes/swan-lake-preview3/RELEASE_NOTE.md

```diff
@@ -2,29 +2,279 @@
+### Overview of Ballerina Swan Lake Preview 3
 
-You can use the update tool to update to jBallerina {{ site.data.stable-latest.metadata.version }} as follows.
+This release is the third preview version of Ballerina Swan Lake. This release includes a new set of language features along with improvements and bug fixes to the compiler, runtime, standard libraries, and developer tooling.
+
+You can use the update tool to update to Ballerina Swan Lake Preview 3 as follows.
 
 **For existing users:**
 
-If you are already using jBallerina version 1.2.0, or above, you can directly update your distribution to jBallerina {{ site.data.stable-latest.metadata.version }} by executing the following command:
+If you are already using jBallerina, you can directly update your distribution to the Swan Lake channel using the [Ballerina Update Tool](http://ballerina.io/swan-lake/learn/keeping-ballerina-up-to-date/). To do this, first, execute the command below to get the Update Tool updated to its latest version. 
+                        
+> `ballerina update`
 
-> $ ballerina dist update
+ Next, execute the command below to update to Swan Lake Preview 3.
 
-However, you need to use the following commands instead of the above if you have installed:
+ > `ballerina dist pull slp3`                  
 
-- jBallerina 1.2.0 but switched to a previous version: `$ ballerina dist pull jballerina-{{ site.data.stable-latest.metadata.version }}`
-- a jBallerina version below 1.1.0: install via the [installers](https://ballerina.io/downloads/)
+However, if you are using a jBallerina version below 1.1.0, install via the [installers](https://ballerina.io/downloads/).
 
 **For new users:**
 
 If you have not installed jBallerina, then download the [installers](https://ballerina.io/downloads/) to install.
 
+### Highlights
+
+- Support for defining external object method bodies
+- Support for executing stored procedures in the SQL connector
+- Azure Functions support
+
+### What's new in Ballerina Swan Lake Preview 3
+
+#### Language
+
+The language implementation is based on [Ballerina Language Specifications Draft 2020-06-18](https://ballerina.io/spec/lang/draft/v2020-06-18/).
+
+##### External object method bodies
+
+This release introduces support for defining object methods with external function bodies.
+
+```ballerina
+type Person object {
+    string fname;
+    string lname;
+
+    function init(string fname, string lname) {
+        self.lname = lname;
+        self.fname = fname;
+    }
+
+    function getFullName() returns string = @java:Method {
+        class: "abc.Hello",
+    } external;
+};
+
+The Java method to which the `getFullName()` method is bound:
+
+```java
```

Comment:
This is not rendering properly

Reply:
Fixed

Reply:
Seems the fix has screwed up it further. Please check the rendering properly

Reply:
Fixed.